requires wave 4 with caveat (#24)
AdaGold/js-adagrams: README.md

#### Process

This Wave has 3 parts:

1. Adjust the syntax for the tests for `highestScoreFrom` to run, instead of skip. To do this, find the `describe` block for the tests of `highestScoreFrom`, and change the syntax from `describe.skip` to `describe`
2. Write the pseudocode for this function, using whatever resources and references
3. Translate the pseudocode into JavaScript, using whatever resources and references

> #### What if I Might Not Finish This Wave?

Reviewer: Maybe, "What If I don't Finish This Wave?" instead of "might not"

You are encouraged the leave the pseudocode as comments for this wave if you begin this wave and don't finish it. The goal for this wave is to practice translating logic to new JavaScript syntax.

> If reviewers see your practice and thought process with pseudocode, they will review accordingly with respect to the practice and work put into the project overall, even if this wave isn't finished.

Reviewer: Maybe add, "If you aren't able to complete this part, please reflect on you process and where you got stuck in the reflection (comprehension?) questions when you submit to project"

> If reviewers see your practice and thought process with pseudocode, they will review accordingly with respect to the practice and work put into the project overall, even if this wave isn't finished.

Reviewer 2: ```suggestion
Our primary goal here is to see your practice and thought process with pseudocode, and we will review accordingly with respect to the practice and work put into the project overall, even if this wave isn't finished.
```

Made it a little softer.  Otherwise looks good.  (I like "we" instead of "reviewers" but that's a personal thing.)
